dotnet/src/SemanticKernel.Abstractions/Data/RecordDefinition/DistanceFunction.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Data;
+
+/// <summary>
+/// Defines the distance functions that can be used to compare vectors.
```

Comment:
This defines well known distance functions but there is nothing to stop an implementor for defining their own, is that correct?

Reply:
Yes, that's correct.  I'll reword to clarify.